### PR TITLE
docs: update `Installation` section in `README.md`

### PR DESCRIPTION
**Pull request by homksei**
## Description

- Updated `Installation` section with another ways

## Solved issues
- https://github.com/uxlfoundation/open-source-working-group/issues/43#issue-2202438296

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,9 +38,19 @@ Deprecation Notice: The Java interfaces are removed from the oneDAL library.
...
+Check the [System Requirements](https://oneapi-src.github.io/oneDAL/system-requirements.html) before installing to ensure compatibility with your system.
+
+There are several options available for installing oneDAL:
+
+- **Binary Distribution**: You can download pre-built binary packages from the following sources:
+    - Intel® oneAPI:
+        - Download as Part of the [Intel® oneAPI Base Toolkit](https://www.intel.com/content/www/us/en/developer/tools/oneapi/onedal.html#gs.8xrue2)
+        - Download as the Stand-Alone [Intel® oneAPI Data Analytics Library](https://www.intel.com/content/www/us/en/developer/tools/oneapi/onedal.html#gs.8xrue2)
+    - [Anaconda](https://anaconda.org/intel/daal4py) (as daal4py)
+    - [NuGet](https://www.nuget.org/packages/Microsoft.ML.OneDal)
```
Comment (homksei) [inline README.md:50]: @napetrov, do we need these options here ?

Comment (napetrov) [reply README.md:50]: yes, but let's put link to base nuget package for onedal, not one that microsfot creating currently 
```suggestion
    - [Anaconda](https://anaconda.org/intel/daal4py) (as daal4py)
    - [NuGet](https://www.nuget.org/packages/inteldal.devel.linux-x64)
```

**Comment by vbm23** [comment]
Hi Team @napetrov @Vika-F @maria-Petrova @aepanchi @Alexsandruss , can anyone help review in order for the same to be merged based on request from @homksei as we ought to meet the timeline for June 28th for https://github.com/uxlfoundation/open-source-working-group/issues/43#issue-2202438296

**Review by Alexsandruss — APPROVED** [review]

**Comment by vbm23** [comment]
Thanks @Alexsandruss 
Team, once this is merged will enable to close #43

**Review by napetrov — APPROVED** [review]

**Pull request merged**